File: C#/leetcode/1768.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace leetcode
{
    internal class _1768
    {
        public string MergeAlternately(string word1, string word2)
        {
             StringBuilder res = new StringBuilder();
            int MinLength = Math.Min(word1.Length, word2.Length);
            for (int i = 0; i < MinLength; i++)
            {
                res.Append(word1[i]);
                res.Append(word2[i]);

            }
            res.Append(word1.Substring(MinLength));
            res.Append(word2.Substring(MinLength));

            return res.ToString();
        }
    }
}

File: Java/leetcode/_1768.java
package leetcode;

class _1768 {

    public String mergeAlternately(String word1, String word2) {
        int minLength = Math.min(word1.length(), word2.length());
        StringBuilder Result = new StringBuilder();
        char[] word1List = word1.toCharArray();
        char[] word2List = word2.toCharArray();

        for (int i = 0; i < minLength; i++) {
            Result.append(word1List[i]);
            Result.append(word2List[i]);
        }
        if(word1.length()>minLength){
            Result.append(word1List, minLength , word1.length()-minLength);
        }
        if(word2.length()>minLength){
            Result.append(word2List, minLength , word2.length()-minLength);
        }
        return Result.toString();
    }
}

//class Solution {
//    public String mergeAlternately(String word1, String word2) {
//        StringBuilder result = new StringBuilder();
//        int minLength = Math.min(word1.length(), word2.length());
//        int i = 0;
//        while (i < minLength) {
//            result.append(word1.charAt(i));
//
//            result.append(word2.charAt(i));
//
//            i++;
//        }
//        result.append(word1.substring(minLength));
//        result.append(word2.substring(minLength));
//        return result.toString();
//    }
//}